tpl/math/math.go

```diff
@@ -72,6 +72,18 @@ func (ns *Namespace) Log(a interface{}) (float64, error) {
 	return math.Log(af), nil
 }
 
+// Sqrt returns the square root of a number.
+// NOTE: will return for NaN for negative values of a
+func (ns *Namespace) Sqrt(a interface{}) (float64, error) {
+	af, err := cast.ToFloat64E(a)
+
+	if err != nil {
+		return 0, errors.New("Sqrt operator can't be used with non integer or float value")
```

Comment:
This error scenario needs some work.  The only two reasons `cast.ToFloat64E` will fail are for a malformed string (using `strconv.ParseFloat`) or for unknown/unsupported data types.  I would suggest something like:

```go
af, err := cast.ToFloat64E(a)
if err != nil {
    return 0, fmt.Errorf("Sqrt operator error: %s", err)
}
```

Also, `cast.ToFloat64E` will allow negative values.  It would probably help our users if we caught negative values before calling `math.Sqrt`.  So, I'd add a separate if-statement before the final return.

```go
if af < 0 {
    return 0, errors.New("Sqrt operator must be greater than or equal to zero")
}
```

I just noticed that `Log` has similar issues.  Let's fix `Log` in a separate PR.